raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -181,10 +234,11 @@ def test_mediated_transfer_with_entire_deposit(
             assert_succeeding_transfer_invariants,
             token_network_address,
             app0,
-            deposit * 2,
+            reverse_calculation.amount_with_fees - reverse_calculation.mediators_cut[0],
             [],
             app1,
-            0,
+            # Why *2 here?
```

Comment:
Do we have an answer for that?

Reply:
yes, it was a misunderstanding on my side when I wrote the comment. Essentially it's forward_fees + reverse_fees which happened to be the same as reverse_fees * 2.